ansible/userdata/cloud-config-etcd

```diff
@@ -40,6 +40,36 @@ coreos:
   update:
     reboot-strategy: "off"
   units:
+    - name: al-agent-container.service
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=Alert Logic Container Service
+        After=docker.service
+        Requires=docker.service
+        [Service]
+        TimeoutStartSec=0
+        Restart=always
+        ExecStartPre=-/usr/bin/docker stop al-agent-container
+        ExecStartPre=-/usr/bin/docker rm al-agent-container
+        ExecStartPre=/usr/bin/docker pull alertlogic/al-agent-container:latest
+        ExecStart=/usr/bin/docker run \
+          --name=al-agent-container \
+          --label "app=al-agent-container" \
+          --net=host \
+          --privileged \
+          --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
+          --mount type=bind,source=/proc,target=/host/proc \
+          --memory-reservation=100m \
+          --memory=500m \
+          --cpus=1 \
```

Comment:
It really needs 1 CPU?

Reply:
This is what was recommended by Alert logic themselves. To stick with a minimum CPU of 1.

Reply:
Though i guess since the etcd instance has just 2 cpus, we can try restricting it further and monitor it

Reply:
What machines do they think we are running these on? `t3.medium` has 2 vCPU, are we supposed to dedicate half of computing power to AlertLogic?

Reply:
the general recommendation was 1. But i guess we can always reduce it 🤷‍♀️

Reply:
done @tamas-molnar